webapp/src/githubprovider.tsx

```diff
@@ -189,7 +189,7 @@ export class GithubProvider extends cloudsync.ProviderBase {
         core.showLoading(LOAD_ID, lf("validating GitHub token..."));
         return Promise.resolve()
             .then(() => {
-                if (hextoken.length != 40 || !/^[a-fA-F0-9]+$/.test(hextoken)) {
+                if (hextoken.length != 40 || !/^gh[pousr]_[a-z0-9_]+$/i.test(hextoken)) {
```

Comment:
I suggest removing the validation check entirely. Let GitHub validate it and we handle the result code accordingly.

Reply:
Seems good to me!